librosa/display.py

```diff
@@ -964,6 +966,12 @@ def __scale_axes(axes, ax_type, which):
         mode = "log"
         kwargs[base] = 2
 
+    elif ax_type == "fft_note":
```

Comment:
This case is identical to `log` (line 959 above) -- should we just merge them like we have for the cqt types?  This would also catch the presently missing `fft_svara` scaling.